pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -274,6 +275,20 @@ type BindingSpec struct {
 	// Names of subordinate objects to create
 	SecretName string `json:"secretName"`
 
+	// TODO: replace with PodPresetTemplateSpec
+
+	// The name of a PodPreset to create.  Must be specified with
```

Comment:
@luksa 

No - they have to be specified together.  The spec only tells you what should be in the `PodPreset`, you still have to give it a name.  We will have a `PodPresetTemplateSpec` type that combines the spec with an `ObjectMeta` so you can control annotations and labels on the created `PodPreset` - see kubernetes/kubernetes#44791.